packages/p2p/src/components/advertiser-page/advertiser-page.scss

```diff
@@ -91,38 +94,120 @@
                 border-right: 1px solid #ededed;
             }
         }
+
+        @include mobile {
+            width: 110vw;
+        }
+
+        &--wrapper {
+            display: flex;
+            align-items: center;
+        }
+
+        .dc-themed-scrollbars__autohide--is-hovered::-webkit-scrollbar-thumb {
+            display: none;
+        }
+    }
+    &__cell {
+        @include mobile {
+            display: block;
+        }
+        &-limit {
+            padding-top: 0.8rem;
+        }
     }
     &__popover {
         &-icon {
-            margin: 1.6rem 0.8rem;
+            margin: 1.6rem 0.8rem 1.6rem 2.4rem;
+
+            @include mobile {
+                margin: 1.6rem 1rem;
+            }
         }
     }
     &__adverts {
         &-button {
             align-items: center;
             display: flex;
             justify-content: center;
+
+            @include mobile {
+                justify-content: flex-end;
+
+                .dc-btn__small {
+                    height: 4rem;
+                }
+            }
         }
+
         &-price {
-            color: var(--text-profit-success);
-            font-weight: bold;
+            @include mobile {
+                margin: 0.2rem 0;
+            }
         }
+
         &-table {
             width: 657px;
         }
+
         &-table_row {
             grid-template-columns: 3fr 2fr 1fr;
+
+            @include mobile {
+                grid-template-columns: repeat(2, 1fr);
+                height: 100px;
+                padding: 1.6rem;
+                width: 100vw;
+            }
         }
+
         &-tabs {
             width: 228px;
         }
+
+        .dc-tabs__list--header-fit-content {
+            @include mobile {
+                width: 100vw !important;
+                display: flex;
+                justify-content: space-evenly;
+                margin-bottom: 0 !important;
+            }
+        }
```

Comment:
Instead of adding CSS rules here, can we add them to the `<Tabs>` component itself? Other pages will 100% need this same mobile state, so it'd be worth doing so.

Reply:
Awesome, this is done.